custom/icds_reports/ucr/data_sources/tasks_cases.json

```diff
@@ -404,6 +404,7 @@
     },
     "engine_id": "icds-ucr-non-dashboard",
     "sql_column_indexes": [
+      // primarily for MPR reports
```

Comment:
json doesn't support comments

Reply:
Oh, right. I will remove it. 

It will be good to keep track of why an index exists on some of the UCRs, though. Not sure where else is good to document this.